core/src/main/java/com/scalar/db/api/Admin.java

```diff
@@ -378,19 +378,19 @@ void repairTable(
    * Add a new column to an existing table. The new column cannot be a partition or clustering key.
    * <br>
    * <br>
-   * <strong>Warning :</strong> this should be executed with significant consideration as the
-   * execution time may vary greatly depending on the underlying storage. Please plan accordingly
-   * especially if the database runs in production:
+   * <strong>Attention:</strong> You should carefully consider adding a new column to a table because the
```

Comment:
@josh-wong We need to run the formatter on the Javadoc for this code. May I push a commit for this?

Reply:
Ran the formatter in 45aa409330c6b2731e347c18b300a744a5c9caee.